plugins/available/pretty-cat.plugin.bash

```diff
@@ -0,0 +1,21 @@
+cite about-plugin                                                                                        
+about-plugin 'pygmentize instead of cat to terminal if possible'                                         
+
+if [ -z $(which pygmentize) ]
+then
+    echo "Pygments is required to use this pluging"
+    echo "Install it by doing 'pip install Pygments' as the superuser"
+    exit 1
+fi
+
+# get the actual cat binary
+CAT_BIN=$(which cat)
+
+# replace the cat binary for a pygmentize if possible
+cat()
+{
+    for var;
```

Comment:
Could you add documentation to this function using the `group`, `about`, `param` and `example` calls like here: https://github.com/Bash-it/bash-it/blob/master/plugins%2Favailable%2Fjava.plugin.bash#L5?